keyboards/crkbd/keymaps/hvp/keymap.c

```diff
@@ -25,56 +26,56 @@ enum custom_keycodes {
 };
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-  [_QWERTY] = LAYOUT( \
+  [_QWERTY] = LAYOUT(
   //,-----------------------------------------.                ,-----------------------------------------.
-     KC_TAB,  KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,                   KC_Y,  KC_U,  KC_I,  KC_O,  KC_P,KC_BSPC,\
+     LT(_ADJUST,KC_TAB),  KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,                   KC_Y,  KC_U,  KC_I,  KC_O,  KC_P,KC_BSPC,
   //|------+------+------+------+------+------|                |------+------+------+------+------+------|
-    LT(_ADJUST, KC_ESC),  KC_A,  KC_S,  KC_D,  LT(_RAISE,KC_F),  MT(MOD_LCTL,KC_G), KC_H,  KC_J,  KC_K,  KC_L,TD(TD1),TD(TD2),\
+    MT(MOD_LSFT,KC_ESC),  KC_A,  KC_S,  KC_D,  LT(3,KC_F), KC_G, KC_H,  KC_J,  KC_K,  KC_L,TD(TD1),TD(TD2),
```

Comment:
You can use `LSFT_T(KC_ESC)` here, to be slightly less verbose

Reply:
Updated